nomos-libp2p/src/behaviour/nat/state_machine/states.rs

```diff
@@ -0,0 +1,210 @@
+use std::fmt::Debug;
+
+use libp2p::Multiaddr;
+
+impl super::State<Uninitialized> {
+    pub const fn new() -> Self {
+        Self {
+            state: Uninitialized(()),
+        }
+    }
+}
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct Uninitialized(
```

Comment:
Can you add a brief comment on top of each state describing what state they actually represent?